.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: |
+          yarn global add lerna
+          # approach taken from https://github.com/actions/setup-node/issues/85
+          echo "$(yarn global bin)" >> $GITHUB_PATH
+      - run: lerna bootstrap
+      - run: lerna run test
+
+name: Build
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    needs: [test]
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install
+      - run: yarn build:dev-ui
+
+name: Deploy Developer UI
+on:
+  push:
+    branches:
+      - master
+      - develop
+    # https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths
+    paths:
+      - "packages/safe-apps-developer-ui/**"
+  pull_request:
+    branches:
+      - master
+  # Launches build when release is published
+  release:
+      types: [published]
+jobs:
+  deploy:
+    name: Deploy
+    needs: [build]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Download website
+        uses: actions/download-artifact@v2
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: 'Deploy to S3: Develop'
+        if: github.ref == 'refs/heads/develop'
+        run: aws s3 sync build/safe-apps-developer-ui s3://${{ secrets.AWS_DEV_BUCKET_NAME }}/safe-apps-developer-ui --delete
```

Comment:
here we should set the correct directory to upload to S3

Reply:
Would the correct path be `packages/safe-apps-developer-ui/build` ? @mikheevm @germartinez

Reply:
I've updated the CI workflow to upload packages/safe-apps-developer-ui/build to S3, could someone double check and merge? @lukasschor

Reply:
Yes the path is correct